core/system/inc/linker.h

```diff
@@ -121,6 +121,9 @@ typedef struct {
     UvisorLibHooks const * const lib_hooks;
 
     TUvisorDebugDriver const * const debug_driver;
+
+    /* Stack limit for MSPLIM_NS */
+    uint32_t * stack_limit;
```

Comment:
1. same here what stack limit is it used for?
2. Additional question - what will we do with this new field on ARMv7-M platforms?
3. Please note the naming conventions here - members are named exactly as in the `uvisor-input.S` except `__uvisor_` prefix